processing/src/main/java/org/apache/carbondata/lcm/status/SegmentStatusManager.java

```diff
@@ -306,13 +330,17 @@ private Integer compareDateValues(Long loadValue, Long userValue) {
    * @return
    */
   public List<String> updateDeletionStatus(String loadDate, String tableFolderPath,
-      Long loadStartTime) {
-    ICarbonLock carbonLock = CarbonLockFactory
+      Long loadStartTime, String dbName, String tableName) throws Exception {
```

Comment:
same comment as above...take the database and table name from carbon table identifier instance